client/resana_secure/gui.py

```diff
@@ -205,7 +211,41 @@ def _on_file_rejected(self, file_path: FsPath) -> None:
             f"Le fichier `{file_path}` a été détecté comme malicieux. Il ne sera pas synchronisé.",
         )
 
+    def _on_conformity_fail(self) -> None:
+        self.message_requested.emit(
+            "Connexion refusée",
+            "L'agent TGB n'est pas joignable.",
+        )
+
+    def _on_conformity_antivirus_fail(self) -> None:
+        self.message_requested.emit(
+            "Connexion refusée",
+            "Votre antivirus ne respecte pas les exigences de conformité.",
+        )
+
+    def _on_conformity_firewall_fail(self) -> None:
+        self.message_requested.emit(
+            "Connexion refusée",
+            "Votre parefeu ne respecte pas les exigences de conformité.",
```

Comment:
Idem : je mettrais un lien vers les exigences, ou vers la doc en ligne de TGB pour savoir ce qui est compliant. 

Aussi, pare-feu c'est avec tiret il me semble
```suggestion
            "Votre pare-feu ne respecte pas les exigences de conformité.",
```